### ISSUE TITLE
wcag22aa tag showing No violations

### ISSUE THREAD

**Issue by sethims** [component-driven/cypress-axe#179]
Product
axe-core

Product Version
4.10.0

Latest Version
 I have tested the issue with the latest version of the product
Issue Description
Expectation
I am trying to use the tag wcag22aa to test accessibility violations against WCAG 2.2 AA standard.

Actual
My Test case is getting passed with No Violation Found

How to Reproduce
using below two lines after visiting the screen where I need to verify accessibility-
cy.injectAxe();
cy.checkA11y(null, {runOnly: {type: "tag",values: ["wcag22aa"]}});

**Comment by sethims** [comment]
@NicholasBoll  Could you please look into this?
